infra/prod/_modules/cosmos/cosmos_container-client.tf

```diff
@@ -0,0 +1,39 @@
+resource "azurerm_cosmosdb_sql_container" "client" {
+
+  name                = "client"
+  resource_group_name = var.resource_group_name
+  account_name        = module.cosmosdb_account_fims.name
+  database_name       = module.cosmosdb_database_fims.name
+
+  partition_key_path    = "/organizationId"
+  partition_key_version = 2
```

Comment:
what's this value for?

Reply:
`2` stands for "large partition keys", which definition is explained [here](https://learn.microsoft.com/en-us/azure/cosmos-db/large-partition-keys?tabs=dotnetv3).
I copied the value from the current Cosmos account, assuming the development team already evaluated it.
Any thoughts? cc @lucacavallaro @michaeldisaro

Reply:
It can be removed, I think it was a typo.

Reply:
(it can be removed everywhere)

Reply:
it looks like the docs recommend it anyway, maybe we should put this value into the module (as default)